measures/SimulationOutputReport/measure.rb

```diff
@@ -489,6 +494,41 @@ def get_cost_multiplier(cost_mult_type, model, runner, conditioned_zones)
             cost_mult += OpenStudio::convert(max_value, "W", "kBtu/h").get
         end
         
+    elsif cost_mult_type == "Size, Heating Supplemental System (kBtu/h)"
+    
+        # Unitary system?
+        if component.nil?
```

Comment:
I think there is no error because this always comes after the "Size, Heating System (kBtu/h)" block, in which `component` is defined. I will add.

Reply:
That shouldn't matter because component is only defined inside the method. Each time the method is called, `component` shouldn't be defined. In fact, I went into the measure test and commented out everything other than the supplemental heating capacity check and it still worked without error... 

❓ ❓ ❓